src/cli.ts

```diff
@@ -189,12 +189,13 @@ export function getCli() {
         });
     }, (argv) => {
       const apiKey: string = <string>argv.apiKey; // required
+      const address: string = <string>argv.address; // required
```

Comment:
good idea, would be cool if it pulled constructor args from etherscan/ chain too. is that a thing?

Reply:
I think the concern there is the code to pull constructor args from Etherscan is a little hard-- though that's a good point, since we use a scraper to do it for `import`. I think after this is done, we should take a look from 500 ft at the lessons we learned from `networks` config and `saddle` and all that and figure out where we *want* to be. Saddle was supposed to be slim and nice, but now it does everything, just like truffle.

Reply:
I *think* if/since we know the deployment block we could pull it off the end of the tx input data